kitsune/sumo/jinja2/base.html

```diff
@@ -315,7 +315,7 @@ <h5 class="mzp-c-footer-heading">{{ _('Firefox Accounts') }}</h5>
   </div>
 </footer>
 {% if request.LANGUAGE_CODE %}
-  {% if settings.DEV %}
+  {% if settings.DEV or settings.TEST %}
```

Comment:
you did mention in our chat the reason this is used but could you please go through it again? short memory - sorry :)

Reply:
Sure: it's to avoid having to minify the django l10n files before running the unit tests in CI - it saves us a few seconds on each run.